### DISCUSSION TITLE
GitHub Actions is currently disabled for your account

### DISCUSSION THREAD

**Discussion by arch-herobrine** [community/community#49465] [Actions] [answered]
### Select Topic Area

Question

### Body

![image](https://user-images.githubusercontent.com/112319383/223597426-7dc58e4e-3168-4e74-b8a4-91e71a7f430e.png)

**Comment by Akash1134** [comment, marked as answer]
Hi @arch-herobrine , Please get in touch with our [Support team](https://support.github.com/contact) and open a ticket with them. In the interest of security, we cannot share account-specific details here. Thanks!

**Reply by shubhankartrivedi** [reply]
Yeah that's all what we did. But imagine waiting for a year to get a reply on support ticket and still don't get any reply......

Its like the whole GitHub support is dead right now....
Was about to buy GitHub pro but this feels very bad....

**Reply by localagi** [reply]
+1 - no the smallest hint why. Support ticket opened.
How long you think you can wait for your builds when on active development?
Already few hours seem too long. When i read about a year I can close my account rightaway.

I'm pro btw, so even paid for the service

**Reply by shubhankartrivedi** [reply]
Dang yeah. I would have closed too but my account is very old and have soo many repos and commits.

**Reply by localagi** [reply]
just leaving this here..

```
gh api graphql --paginate -f query='
    query($endCursor: String) {
      viewer {
        repositories(first: 100, after: $endCursor) {
          nodes { nameWithOwner }
          pageInfo {
            hasNextPage
            endCursor
          }
        }
      }
    }

'| jq '..|.nameWithOwner?' | sed '/null/d' | xargs -I {} git clone https://github.com/{}

```

**Reply by ghost** [reply]
plz help my account's GitHub Actions got disabled for no reason

**Comment by hasnatulhassan** [comment]
Hi, my repository action currently disable Notice: "GitHub Actions is currently disabled for your account. Please reach out to [GitHub Support] for assistance."

**Comment by zaakir10** [comment]
GitHub Actions is currently disabled for your account.  Pls help this problem

**Comment by robinnik1** [comment]
help

**Comment by moexalter** [comment]
my repository action currently disable Notice: "GitHub Actions is currently disabled for your account. Please reach out to [GitHub Support] for assistance.

**Reply by robinnik1** [reply]
contact support

**Comment by ghost** [comment]
plz help my account's GitHub Actions got disabled for no reason

**Reply by robinnik1** [reply]
contact support and what repo did you made

**Comment by ghost** [comment]
Help how to fix this

**Reply by Sivamani-18** [reply]
To resolve the issue of GitHub Actions being disabled for your account, you can follow these steps:

1. **Contact GitHub Support:**
   - Go to the [GitHub Support page](https://support.github.com).
   - You might need to sign in to your GitHub account if you haven't already.
   - Once you're on the support page, look for the option to submit a request or report a problem.
   - Explain your issue clearly, including that GitHub Actions are disabled for your account and you need assistance to enable them.

2. **Provide Necessary Information:**
   - When submitting your request, provide any relevant details such as your GitHub username, repository name, and any actions you've taken that led to this message.
   - Be clear and concise in describing the problem.

3. **Follow Up:**
   - After submitting your request, you should receive a confirmation email from GitHub Support.
   - Keep an eye on your email for any responses or additional instructions from the support team.
   - Respond promptly to any follow-up questions or requests for more information.

4. **Check GitHub Status and Documentation:**
   - Occasionally, there may be broader issues affecting GitHub Actions. Check the [GitHub Status page](https://www.githubstatus.com) to see if there are any ongoing incidents.
   - Review the [GitHub Actions documentation](https://docs.github.com/en/actions) to ensure there are no specific rules or changes that might apply to your account.
